go.mod

```diff
@@ -122,4 +122,4 @@ require (
 )
 
 //replace github.com/spectrocloud/hapi => ../hapi
-//replace github.com/spectrocloud/palette-sdk-go => ../palette-sdk-go
+replace github.com/spectrocloud/palette-sdk-go => /Users/nik/wsp/spectro/palette-sdk-go
```

Comment:
is this intenionaly added

Reply:
@SivaanandM going to remove after SDK change is done.